scripts/gen_openapi.sh

```diff
@@ -37,7 +37,13 @@ trap defer_stop_vault INT TERM EXIT
 
 export VAULT_ADDR=http://127.0.0.1:8200
 
-echo "Mounting all builtin plugins..."
+echo "Unmounting the default kv-v2 secrets engine ..."
+
+# Unmount the default kv-v2 engine so that we can remount it at 'kv_v2/' later.
```

Comment:
Did you deliberately want to switch to underscores here? Hyphens are feel more usual in Vault paths to me, and are what you have used in the PR comment descriptive text, and in the existing openapi.json checked in to the vault-client-go Git repo.

Reply:
It felt weird to have a mixture of underscores and hyphens in `kv-v2_mount_path`. Switching these to underscores made the resulting names more consistent: `kv_v2_mount_path`.

Reply:
Oh now I see that it affects the default mount path values as well. In this case hyphens are probably better.

Reply:
I've changed it to hyphens and added a small change to replace hyphens with underscores in the param names: https://github.com/hashicorp/vault/pull/21563/commits/d580381998640921914b00e5d59bf41b7c2a2add